web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -75,13 +79,37 @@
 
   $: people = asset.people || [];
   $: showingHiddenPeople = false;
+  $: showActionPerson = Array(people.length).fill(false);
 
   const unsubscribe = websocketStore.onAssetUpdate.subscribe((assetUpdate) => {
     if (assetUpdate && assetUpdate.id === asset.id) {
       asset = assetUpdate;
     }
   });
 
+  const handleRightClickPeople = (index: number) => {
+    showActionPerson[index] = !showActionPerson[index];
+  };
+
+  const handleHidePerson = async (index: number) => {
+    const timeout = setTimeout(() => (isShowLoadingUpdatePerson = true), 100);
```

Comment:
Having a timeout seems rather ugly... Does it behave weirdly if we don't wait those 100ms?

Reply:
Yep, if you don't have this timeout, the loading spinner flashes even if you have a low latency.

Reply:
Exactly

Reply:
It's clearly not needed, but I like to show that the request is being processed when you update something.
For exemple, in the case where the user is disconnected and tries to update a person visibility, if he hits the "Hide"/"Unhide" button, he won't be surprised if the person visibility does not change.

Reply:
Fair! Then I'm fine with leaving it as is :)